src/Razor/Microsoft.AspNetCore.Razor.Language/test/TestFiles/IntegrationTests/ComponentRuntimeCodeGenerationTest/ChildComponent_Generic_TypeInference_Cascaded/TestComponent.codegen.cs

```diff
@@ -0,0 +1,66 @@
+// <auto-generated/>
+#pragma warning disable 1591
+namespace Test
+{
+    #line hidden
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Components;
+    public partial class TestComponent : Microsoft.AspNetCore.Components.ComponentBase
+    {
+        #pragma warning disable 1998
+        protected override void BuildRenderTree(Microsoft.AspNetCore.Components.Rendering.RenderTreeBuilder __builder)
+        {
+            {
+                __Blazor.Test.TestComponent.TypeInference.CreateGrid_0_CaptureParameters(
+#nullable restore
+#line 1 "x:\dir\subdir\Test\TestComponent.cshtml"
+               Array.Empty<DateTime>()
+
+#line default
+#line hidden
+#nullable disable
+                , out var __typeInferenceArg_0___arg0);
+                __Blazor.Test.TestComponent.TypeInference.CreateGrid_0(__builder, 0, 1, __typeInferenceArg_0___arg0, 2, (__builder2) => {
+                    __Blazor.Test.TestComponent.TypeInference.CreateColumn_1(__builder2, 3, __typeInferenceArg_0___arg0);
+                    __Blazor.Test.TestComponent.TypeInference.CreateColumn_2(__builder2, 4, __typeInferenceArg_0___arg0);
+                }
+                );
+                __typeInferenceArg_0___arg0 = default;
```

Comment:
Why do we need this statement at the end?

Reply:
The intent is to avoid pinning the captured value indefinitely. The variable will be used within closures and hence would otherwise be pinned for GC for as long as that closure remains reachable. We definitely don't need any object pointed to by the variable, so we can safely blank it out and know we're not unnecessarily pinning its value.